### ISSUE TITLE
map*() and install() functions should be declared as functions not types

### ISSUE THREAD

**Issue by purumpum** [sobolevn/vue-flow-typed#26]
After updating to version 0.40.0 I ran into following error:
``Cannot import the type `mapState` as a value. Use `import type` instead.``.

According to examples in the official documentation, you should declare exported functions in this way:
`declare export function(arg1: arg1Type, ..., argN: argNType): resultType`.

Also, declare actual exported function as a type looks strange:
`import { mapState } from vuex;`
`import type { mapState } from 'vuex';`
`const alias: mapState = mapState; `

**Comment by sobolevn** [comment]
What do you mean by version `0.40.0`? It was released so long ago.
Are you sure that this should be supported?

**Comment by purumpum** [comment]
@sobolevn, sorry, my words were unclear. I have this issue with vue-flow-typed@0.40.0 and the latest versions of Flow (0.89.0, 0.90.0, 0.91.0). I also test my fix with that versions of Flow. If needed, I can provide a sample project to reproduce this issue.

**Comment by sobolevn** [comment]
Oh, I see. Thanks for clarification.

How does this relate to https://github.com/sobolevn/vue-flow-typed/pull/25?
Because it is the reverse change.

I am not using `flow` with `vue` at the moment, so I can not find it out by myself.

**Comment by purumpum** [comment]
My fix is almost a reverse change except for one string:
```diff
- install: install,
+ install: typeof install,
```
I have same issue #24 with `vue-flow-typed@0.3.0` because since `0.82.0` Flow disallows to use function declaration of  `install` named export as a type of the property of default export object. Changing declaration of `install` export (and also `map*` exports) from function to type in #25 makes the libdef valid for Flow but totally unusable due to error I mentioned in the first message.

To fix issue #24 I just extract type of previously declared function by using `typeof` operator. Of course, it is possible to declare InstallFn type to type both `install `named export and a property of default export object or just copy the declaration of `install `named export to default export object:
```diff
- install: install,
+ install: (Vue: any) => void,
```
